classes/XLite/Module/PureClarity/Personalization/View/Customer/Events/Order.php

```diff
@@ -62,6 +63,12 @@ protected function getOrderEventData() : string
         $orderData = [];
         $orderId = Request::getInstance()->order_number;
```

Comment:
Is there anyway to combine this with the new code? If we're pulling this already can we check if it's null? Currently we seem to get Request::getInstance()->order_number and then run the check again which will get it again if we don't have an order id.

We could probably just do 

```
if (Request::getInstance()->order_id) {
            $orderId = Request::getInstance()->order_id;
        }
```

since we already have the elseif set?